.header.md

```diff
@@ -287,6 +287,26 @@ Key points about VPC endpoint types:
 - **Internal endpoint**: Created when `address_allocation_ids` are omitted
 - Internal endpoints are only accessible from within the VPC or connected networks
 
+## Example for Public Endpoint Configuration
+
+This example demonstrates a simple public endpoint configuration:
+
+```hcl
+module "transfer_server" {
+  # Other configurations go here
```

Comment:
same thing over here, if you could explain what this other configuration is or omit if not required